src/extension/wps/doc/GOGODUCK_README.md

```diff
@@ -74,32 +81,33 @@ Run:
 $ mvn test
 ```
 
-## Configuring GoGoDuck
+## Configuring GoGoDuck downloads via the Portal
 
 ### Metadata Record
 
-A typical Geonetwork record for GoGoDuck will include the following online
-resource:
+A typical Geonetwork record to enable a GoGoDuck download via the Portal will include the following online resource:
+
 ```
 <gmd:onLine>
   <gmd:CI_OnlineResource>
     <gmd:linkage>
-      <gmd:URL>GEOSERVER_OWS_URL</gmd:URL>
+      <gmd:URL> [GEOSERVER_OWS_URL] </gmd:URL>
     </gmd:linkage>
     <gmd:protocol>
       <gco:CharacterString>OGC:WPS--gogoduck</gco:CharacterString>
     </gmd:protocol>
     <gmd:name>
-      <gco:CharacterString>LAYER_NAME</gco:CharacterString>
+      <gco:CharacterString> [LAYER_NAME] </gco:CharacterString>
     </gmd:name>
     <gmd:description>
-      <gco:CharacterString>The GoGoDuck subsets and aggregates gridded data.  Data is returned as a NetCDF file</gco:CharacterString>
+      <gco:CharacterString>The GoGoDuck subsets and aggregates gridded data.  Data is returned as a NetCDF file or CSV</gco:CharacterString>
```

Comment:
Yes true. We will endevour to merge when the remaining CSV output work is merged